<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainpage">
     <widget class="QPushButton" name="pushButtonTemp">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Turbldlty</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonpH">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>pH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clickButtonTDS">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>TDS</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTemp2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelTur2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>76</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelTDS2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>350</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelpH2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>76</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pHPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>pH</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="chartWidgetpH" native="true">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>661</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButtonpH">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/image/pH.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelpH">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TurPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Turbldlty</string>
      </property>
     </widget>
     <widget class="QWidget" name="chartWidgettur" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>681</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButtonTur">
      <property name="geometry">
       <rect>
        <x>689</x>
        <y>16</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/image/tur.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelTur">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TempPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QWidget" name="chartWidgettem" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>701</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButtonTemp">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/image/tem.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelTemp">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TDSPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TDS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButtonTDS">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QWidget" name="chartWidgetTDS" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>651</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/image/tds.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelTDS">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
